CONTRIBUTING.md

```diff
@@ -1,5 +1,7 @@
 # Contributing to Simple Icons
 
+> ℹ We ask that all users read our [legal disclaimer](./DISCLAIMER.md) before contributing to Simple Icons.
```

Comment:
One minor request, I'd say we should avoid using `ℹ` directly here (and in the README.md) so as to keep this document "text only" - just in case someone on an old system want to read this.

```suggestion
> :information_source: We ask that all users read our [legal disclaimer](./DISCLAIMER.md) before contributing to Simple Icons.
```